script/gke-e2e.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
```

Comment:
Why adding a new gke-e2e.sh, while lots of common code with e2e-kube.sh?

Reply:
e2e-kube is using espcli.   I want to move away  from espcli.

That is why I create a new script to use the deployment yaml file directly.

for new gke tests, they will be using this new script.